### PR TITLE
Upgrade Decidim to v0.20

### PR DESCRIPTION
**Pull request by LauraJaimeC**
CHANGELOG: https://github.com/decidim/decidim/blob/release/0.20-stable/CHANGELOG.md

- Upgrade Decidim to v0.20

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ source 'https://rubygems.org'
 
 ruby RUBY_VERSION
 
-DECIDIM_VERSION = { git: 'https://github.com/decidim/decidim.git', branch: '0.19-stable' }
+DECIDIM_VERSION = { git: 'https://github.com/decidim/decidim.git', branch: 'release/0.20-stable' }
```
Comment (tramuntanal) [inline Gemfile:7]: use CodiTramuntana's fork

**Review by tramuntanal — CHANGES_REQUESTED** [review]
Hi @LauraJaimeC ,
Please remember to always link the changelog for the corresponding version:
https://github.com/decidim/decidim/blob/release/0.20-stable/CHANGELOG.md

From this changelog, remember that we need to change the geocoder configuration, and apply the searchable indexing as stated here: https://github.com/decidim/decidim/blob/release/0.20-stable/CHANGELOG.md#decidimsearchable

File: Gemfile (hunk 1 above)
Comment (LauraJaimeC) [reply Gemfile:7]: In our fork don't have release 0.20. Do I create the branch?

Comment (LauraJaimeC) [reply Gemfile]: done!

**Comment by LauraJaimeC** [comment]
Closed in favor of https://github.com/CodiTramuntana/decidim-sitges-app/pull/37

**Pull request closed without merging**